src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,13 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  user?: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
+    </a>
```

Comment:
[Checklist №4 ](https://github.com/mate-academy/react_add-todo-form/blob/master/checklist.md) [REACT KNOWLEDGE] - Don't render the component if the property that you pass to the component has null or undefined value.